src/components/attendee/VideoHuddle/useVideoHuddle.tsx

```diff
@@ -1,24 +1,103 @@
-import { useContext } from "react";
+import React, { useContext, useMemo, useState } from "react";
+import { IconDefinition } from "@fortawesome/free-solid-svg-icons";
 
-import { VideoHuddleContext } from "./VideoHuddle";
+import { useVideoComms } from "../VideoComms/hooks";
+import { VideoTrack } from "../VideoComms/types";
+
+interface HuddleContextType {
+  inHuddle: boolean;
+  setInHuddle: (cb: (prevInHuddle: boolean) => void) => void;
+
+  extraButtons: ButtonConfig[];
+  setExtraButtons: (buttons: ButtonConfig[]) => void;
```

Comment:
Shouldn't listeners start with `onInHuddle` and `onSetExtraButtons`? How about `onChildren` instead? (see above)